recipes/quickfix-cpp/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "40b8984" %}
```

Comment:
Before we merge, this must be set to a real version.

Reply:
Yea.. fingers crossed, last official pin was 2018, but the repo is actively still releasing updates/commits - added a request here: https://github.com/quickfix/quickfix/issues/596

Reply:
Updated to '1.15.1.postYYYYMMDD' format, given lack of movement on official tags from package author